cmd/crates/soroban-spec-tools/src/utils.rs

```diff
@@ -1,268 +1,4 @@
-use base64::{engine::general_purpose::STANDARD as base64, Engine as _};
 use hex::FromHexError;
-use std::{
-    fmt::Display,
-    io::{self, Cursor},
-};
-
-use stellar_xdr::curr::{
-    Limited, Limits, ReadXdr, ScEnvMetaEntry, ScMetaEntry, ScMetaV0, ScSpecEntry, ScSpecFunctionV0,
-    ScSpecUdtEnumV0, ScSpecUdtErrorEnumV0, ScSpecUdtStructV0, ScSpecUdtUnionV0, StringM,
-};
-
-pub struct ContractSpec {
```

Comment:
Is the code in this file being deleted because it was already unused? We'll need to keep that deletion for a major version. Can you open an issue for it and label the issue with `breaking` so we know to come back and do it before then?

Reply:
Yeah it was unused, it's copy-paste of [contract-rs ](https://github.com/stellar/stellar-cli/blob/56a81bedbbc33300fa5e2d933808a5bbd7b7b875/cmd/crates/soroban-spec-tools/src/contract.rs)

Reply:
Ah, it isn't even exported from the crate? Then it is fine to delete.

Reply:
> pub mod utils;
> pub struct ContractSpec

I think it's exported 🤔

Reply:
https://github.com/stellar/stellar-cli/issues/1544